src/db.rs

```diff
@@ -467,11 +581,83 @@ mod tests {
         ];
 
         let ksetid = Uuid::new_v4().as_hyphenated().to_string();
-        let polid = 1;
-        insert_keyset(&ksetid, &keys, Some(polid))?;
-        let keyset_ids = get_keyset_ids(&ksetid)?;
+        let polid: Option<u32> = Some(1);
+        aw!(insert_keyset(&ksetid, &keys, polid))?;
+
+        let keyset_ids = aw!(get_keyset_ids(&ksetid))?;
         assert_eq!(keyset_ids.len(), keys.len());
         assert_eq!(keyset_ids, keys);
+
+        aw!(delete_keyset(&ksetid))?;
+
+        Ok(())
+    }
+
+    #[test]
+    fn test_get_report_keypair() -> anyhow::Result<()> {
+        let tid = "man-moon-dog-face-in-the-banana-patch".to_string();
+
+        let rng = SystemRandom::new();
+        let pkcs8_bytes = signature::EcdsaKeyPair::generate_pkcs8(
+            &signature::ECDSA_P256_SHA256_ASN1_SIGNING,
+            &rng,
+        )
+        .unwrap();
```

Comment:
Can you explain why we use actual signature generation in this test, instead of:

```rust
let pkcs8_dummy_bytes = [0xa5u8; 512];
```

since the `db.rs` module doesn't deal with the signatures themselves, I'd simplify its tests to store meaningless binary blobs.

Also I noticed that column `keypair` is `varchar(1024)`, I assume storing base64 strings. So actual binary value is limited to 1024 / 4 * 3 = 768 bytes. That should be OK for EC signatures but let's double-check (CC @fitzthum ).

Reply:
I made this change.  Originally, I wanted to stay as close to a real value as possible, but I take your point.